StemKratos/StemApplication/set_nodal_parameters_process.py

```diff
@@ -0,0 +1,92 @@
+import KratosMultiphysics
+import KratosMultiphysics.StructuralMechanicsApplication as KSM
+
+# Set available nodal parameter options
+NODAL_PARAMETER_OPTIONS = [KSM.NODAL_DAMPING_RATIO, KSM.NODAL_DISPLACEMENT_STIFFNESS,
+                           KSM.NODAL_ROTATIONAL_DAMPING_RATIO,
+                           KSM.NODAL_ROTATIONAL_STIFFNESS, KratosMultiphysics.NODAL_MASS]
+
+
+class SetNodalParametersProcess(KratosMultiphysics.Process):
+    """
+    This process sets the nodal parameters of the model part. The nodal parameters are set to the values of the
+    properties of the model part.
+
+    Inheritance:
+        - :class:`KratosMultiphysics.Process`
+
+    Attributes:
+        - model_part (KratosMultiphysics.ModelPart): model part containing the elements
+        - settings (KratosMultiphysics.Parameters): settings of the process
+
+
+    """
+
+    def __init__(self, model_part: KratosMultiphysics.ModelPart, settings: KratosMultiphysics.Parameters):
+        """
+        Initialize process
+
+        Args:
+            - model_part (KratosMultiphysics.ModelPart): model part containing the elements
+            - settings (KratosMultiphysics.Parameters): settings of the process
+        """
+        KratosMultiphysics.Process.__init__(self)
+        self.model_part = model_part
+        self.settings = settings
+
+    def ExecuteInitialize(self):
```

Comment:
`    def ExecuteInitialize(self): -> None`

Reply:
there is no return, so we discussed before, to not put -> None as a typehint